lldb/source/Commands/CommandObjectProcess.cpp

```diff
@@ -1271,13 +1271,13 @@ class CommandObjectProcessSaveCore : public CommandObjectParsed {
 
       switch (short_option) {
       case 'p':
-        m_requested_plugin_name = option_arg.str();
+        m_core_dump_options.SetPluginName(option_arg.data());
```

Comment:
@clayborg Should we return here the error here from setting an invalid plugin name?